packages/terra-application-links/src/tabs/_MenuButton.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import IconCaretDown from 'terra-icon/lib/icon/IconCaretDown';
+import styles from './ApplicationTabs.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Whether or not the menu button should be hidden.
+   */
+  isMenuHidden: PropTypes.bool,
+  /**
+   * Whether or not the menu button should be animated with selection.
+   */
+  isSelected: PropTypes.bool,
```

Comment:
I feel a prop name like `isAnimated` seems more appropriate & would reflect the props description